staging/istio/templates/deployment.yaml

```diff
@@ -33,12 +33,7 @@ spec:
             runAsNonRoot: true
           imagePullPolicy: IfNotPresent
           resources:
-            limits:
```

Comment:
Why?

Reply:
Yeah, removing the requests is not a good practice, and certain k8s benchmarks could report it as a WARNING or ERROR. Please, justify why you removed the limits, and revert the requests.

Reply:
oh :D, I just saw that their in the values.yaml 🙄 , all good :)